docs/cpp/config-clang-mac.md

```diff
@@ -110,10 +110,25 @@ Remember, the C++ extension uses the C++ compiler you have installed on your mac
 
    ![Screenshot of helloworld.cpp and play button](images/playbutton/run-play-button.png)
 
-3. Choose **C/C++: clang++ build active file** from the list of detected compilers on your system (you'll only be asked to choose a compiler the first time you run `helloworld.cpp`).
+3. Choose **C/C++: clang++ build and debug active file** from the list of detected compilers on your system.
 
    ![Build task](images/clang-mac/default-build-task.png)
 
+you'll only be asked to choose a compiler the first time you run `helloworld.cpp`. This compiler will be set as the "default" compiler in `tasks.json` file. In case you need to change the default compiler, you can run **Tasks: Configure default build task**. Alternatively you can modify the `tasks.json` file and remove the default by replacing this segment:
```

Comment:
Real quick, can you make the start of the sentence capitalized? "You'll only be asked...."
Same for the sentence in the other topics. Thanks. Otherwise looks good to go